src/work-allocation/components/info-message/info-message.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+import {InfoMessage, InfoMessageType} from './../../enums';
+
+@Component({
+  selector: 'exui-info-message',
+  templateUrl: './info-message.component.html',
+})
+export class InfoMessageComponent implements OnInit {
+
+  /**
+   * The type of message to display. ie. Success, Warning or Information message.
+   */
+  @Input() public infoMessageType: InfoMessageType;
+
+  @Input() public infoMessage: InfoMessage;
+
+  public infoMessageType = InfoMessageType;
+
+  constructor() {}
+
+  public ngOnInit(): void {
```

Comment:
Is something going to go in here at some point in the future? If not, it's not needed. The same for the constructor.

Reply:
Good spot, left in accidentally.